script/Ciel/Day 15/15_01B_CIEL15B_1.txt

```diff
@@ -3150,7 +3150,7 @@ It's like I'm only now realizing just what an incredible person I've fallen in l
 [sha:069c41b235436ce9a7e576c1252efb0f6d60ad4b]{
 -- Page 192, Offset 42531.
 -- けどまあ、それも覚悟の上だ。
-Though, well, I always knew that was part of the package.
+Though, well, I always knew that was all part of the package.
```

Comment:
Yeah still think this one's fine as is.

Reply:
It's as is or the other suggestion rs had before.
"All" here just makes the line read a bit misbalanced.